File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int k = 1;
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1);

    for (int i = 1; i <= h; i++)
    {

        for (k = 1; k <= h - i; k++)
        {
            printf(" ");
        }
        for (int j = 1; j <= h; j++)
        {
            printf("#");
            if (k > h - j)
            {
                break;
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int
while True:
    n = get_int("Height: ")
    if n <= 0 or n > 8:
        pass
    else:
        break
k = 1
for i in range(n):
    for k in range(n-i-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")

    print()
